### PR TITLE
Feature/configs

### PR DESCRIPTION
**Pull request by hashim-sohail**

### CODE REVIEW

File: src/deploy.js [hunk 1]
```diff
@@ -12,6 +12,14 @@ async function deploy() {
 
 	const name = slugify(GITHUB_REPOSITORY_NAME);
 
+	console.log('Final Object', {
```
Comment (artalat) [inline src/deploy.js]: do we need to keep these logs in final version?

File: dist/index.js [hunk 2]
```diff
@@ -8382,6 +8382,14 @@ async function deploy() {
 
 	const name = slugify(GITHUB_REPOSITORY_NAME);
 
+	console.log('Final Object', {
```
Comment (artalat) [inline dist/index.js:8385]: do we need to keep these logs in final version?

File: README.md [hunk 3]
```diff
@@ -19,10 +19,13 @@ This action uses GitHub API to set published build as a deployment.
 The url of the client bundle.
 
 ## Example usage
+You can pass all [NOW configurations](https://zeit.co/docs/v2/advanced/configuration) as JSON String to the configs variable
 
 ```yml
 - name: Now Deploy
   uses: BlueBaseJS/now-deploy-github-action@master
+  with:
```
Comment (artalat) [inline README.md:27]: Since this is bluebase org, maybe use bluebase example?

**Review by artalat — CHANGES_REQUESTED** [review]

File: src/deploy.js (hunk 1 above)
Comment (hashim-sohail) [reply src/deploy.js]: No, we don't need them, just added to verify that configs were being passed.

File: README.md (hunk 3 above)
Comment (hashim-sohail) [reply README.md:27]: I'll edit the readme

File: dist/index.js (hunk 2 above)
Comment (hashim-sohail) [reply dist/index.js:8385]: Removed

**Review by artalat — APPROVED** [review]

**Pull request merged**